blockchain/vm/interface.go

```diff
@@ -104,4 +104,6 @@ type StateDB interface {
 	GetTxHash() common.Hash
 
 	GetKey(address common.Address) accountkey.AccountKey
+
+	Finalise(bool, bool)
```

Comment:
If we move ProcessParentBlockHash into consensus directory, check if we still need this addition. engine.Initialize receives the concrete type *StateDB, not this interface vm.StateDB.

Reply:
Waiting for the [prerequisit](https://github.com/kaiachain/kaia/pull/121#discussion_r1825432048)